extension/content_script.js

```diff
@@ -518,6 +522,7 @@ function oneTimeSetup() {
 			}
 		);
 	});
+	cb();
```

Comment:
no ben we talked about this

-- jk or maybe you meant cb() _after settings is loaded?
another option is to just put setupKeyListenerForOnOff() right after settings.getSpeed

Reply:
First option is what was meant; if "R" is clicked before this is done setting up it causes problems